### PR TITLE
added support for multiple ci environments in repipe

### PR DESCRIPTION
**Pull request by lnguyen**

### CODE REVIEW

File: bin/genesis [hunk 1]
```diff
@@ -3096,15 +3096,6 @@ ci_update() {
 	fi
 }
 
-ci_pipeline_yaml() {
```
Comment (dennisjbell) [inline bin/genesis]: why'd you get rid of this?

Comment (lnguyen) [inline bin/genesis]: cause we have two functions with same name

Comment (jhunt) [inline bin/genesis]: kudos on picking the right one to remove.  the remaining definition is actually correct 🤣

**Comment by jhunt** [comment]
LGTM :+1:

**Pull request merged**